src/runners/handlers/slack.py

```diff
@@ -67,14 +68,18 @@ def handle(
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
 
     sc = SlackClient(slack_token)
-
+    client = WebClient(slack_token)
```

Comment:
could re-use the SlackClient we already have for consistency? I think it can make any API call via `sc.api_call` as used below